### PR TITLE
Update custom domain help

### PR DESCRIPTION
**Pull request by ChristopheDujarric**
Trying to improve his doc a little, after reviewing the latest support requests on the topic.

### CODE REVIEW

File: docs/help/custom-domains.md [hunk 1]
```diff
@@ -1,23 +1,28 @@
 # Custom domains
 
-## Before you begin
+Your docs are hosted on Bump.sh. However you can set a custom domain: visitors of your docs will stay within your brand experience. For this, you'll use a CNAME.
```
Comment (jayk-u) [inline docs/help/custom-domains.md:3]: > your docs will stay within your brand experience

Brand sounds a bit too commercial to me, visitors can also be from the organization.

Comment (paulRbr) [reply docs/help/custom-domains.md:3]: I completely agree with Jay remark. Why do we need to talk about brand here?

Comment (paulRbr) [reply docs/help/custom-domains.md:3]:
```suggestion
Your docs are hosted on Bump.sh. However you can set a custom domain: visitors of your docs will then stay within your own name. For this, you'll use a CNAME.
```

Comment (paulRbr) [reply docs/help/custom-domains.md:3]: I've moved the suggestion to #66

File: docs/help/custom-domains.md [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
 
 1. Click on the chosen **Documentation** **>** **Settings**
 2. Check the **Setup a custom domain **box.
 3. Enter your custom domain.
 4. Click **Update global settings**.
 
+That's it. SSL certificates are automatically issued. 
 
-
+Note: DNS caches may take a moment to update. Outdated caches may lead to the fact that visitors will not be able to access your docs via your custom domain. Those caches are user-side, which means that 1. you can flush it on your local computer 2. people who never visited your doc will not face any issue.
```
[the remark this answers is not in the document]
Comment (paulRbr) [reply docs/help/custom-domains.md:28]: I've moved the suggestion to #66

**Pull request merged**